src/Arcus.EventGrid.Tests.Integration/Publishing/EventPublishingTests.cs

```diff
@@ -36,17 +36,15 @@ public EventPublishingTests(ITestOutputHelper testOutput)
 
         public async Task DisposeAsync()
         {
-            await _hybridConnectionHost.Stop();
+            await _serviceBusEventConsumerHost.StopAsync();
         }
 
         public async Task InitializeAsync()
         {
-            var relayNamespace = Configuration.GetValue<string>("Arcus:HybridConnections:RelayNamespace");
-            var hybridConnectionName = Configuration.GetValue<string>("Arcus:HybridConnections:Name");
-            var accessPolicyName = Configuration.GetValue<string>("Arcus:HybridConnections:AccessPolicyName");
-            var accessPolicyKey = Configuration.GetValue<string>("Arcus:HybridConnections:AccessPolicyKey");
+            var connectionString = Configuration.GetValue<string>("Arcus:ServiceBus:ConnectionString");
+            var topicName = Configuration.GetValue<string>("Arcus:ServiceBus:TopicName");
 
-            _hybridConnectionHost = await HybridConnectionHost.Start(relayNamespace, hybridConnectionName, accessPolicyName, accessPolicyKey, _testLogger);
+            _serviceBusEventConsumerHost = await ServiceBusEventConsumerHost.Start(topicName, connectionString, _testLogger);
```

Comment:
A suggestion made by @samneirinck 	was to initialize via ctor and call Start after that so that we don't need statics.

Reply:
Yes, I'd rather prefer it this way:

```csharp
// This can already throw exceptions if the configuration is wrong
var host = new ServiceBusEventConsumerHost(topicName, connectionString, _testLogger);

// This would only throw exceptions if something went wrong starting it
await host.Start();
```

I think it's also more intuitive, I wouldn't expect a `Start` method to actually return me the object that it created.

Reply:
Created a rough approach with a constructor which you can find here:
https://github.com/tomkerkhove/arcus.eventgrid/pull/1

I'm not really a fan of it to be honest because it creates an object in a mixed state because some parts are only known after the subscription was created which requires async.

Reply:
Discussed this with Stijn offline and given that we're creating initialize objects that are not complete we're going to stick with the current approach for now.